logpackages: Ensure we are back to text console

okurz opened the pull request:
See https://openqa.suse.de/tests/621709#step/logpackages/2

coolo commented:
Please fix https://progress.opensuse.org/issues/14344 while you're at installation consoles

okurz commented:
Updated:
* added explicit assert_screen in `install_and_reboot`, too
* took maintainership of both `logpackages` and `install_and_reboot`

Took ticket [poo#14344](https://progress.opensuse.org/issues/14344)

DimStar77 commented:
Likely responsible for the earlier fail of KDE Live install test: https://openqa.opensuse.org/tests/287789#step/install_and_reboot/5

In the live installer, there is normal TTYs only; none has the special instruction text you'd get when booting the regular installer media

coolo commented:
I guess we shouldn't treat live installations as normal ones.

coolo commented:
(i.e. use a different console name for them)

coolo commented:
https://openqa.suse.de/tests/622303#step/logpackages/4 - this broke s390
